python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -494,11 +503,13 @@ def __set__(self, obj, val):
                 val = self._stripAttribute(obj, val, getName(self))
 
         # apply attribute filter to component items
-        if item.isA(Schema.ComponentItem):
+        if item.isA(ComponentItem):
             val = self._stripAttribute(obj, val, getName(self))
 
         self._check_type(obj, val)
-        setattr(stripProxy(obj), getName(self), val)
+        
+        GangaObject = _getGangaObject()
+        GangaObject.__setattr__(stripProxy(obj), getName(self), val)
```

Comment:
In principle we're not overloading it atm but I don't think we lose anything by using the methods provided by ```GangaObject``` (which are the same as ```object``` for now at least). The redirection is minor but it could allow for us to use this for debugging or other uses in the future imo.